med_inventory/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+
+class Meta:
```

Comment:
Since there is no longer a `Stock` class, I do not think that the `Meta` class needs to stay either

Reply:
This meta class is there to add the permission/capability to check the stock, which then can be applied to the pharmacist group.